site/_docs/algebra.md

```diff
@@ -265,17 +265,7 @@ or `builder.field(2, 1, 1)`.
 ### Recursive Queries
 
 Warning: The current API is experimental and subject to change without notice.
-A SQL recursive query, e.g. this one that generates the sequence 1, 2, 3, ...10:
-
-{% highlight sql %}
-WITH RECURSIVE aux(i) AS (
-  VALUES (1)
-  UNION ALL
-  SELECT i+1 FROM aux WHERE i < 10
-)
-SELECT * FROM aux
-{% endhighlight %}
-
+A recursive query, e.g. this one that generates the sequence 1, 2, 3, ...10,
```

Comment:
I prefer the SQL example here as I find it easier to follow the example afterwards.